src/Copy/Text.elm

```diff
@@ -137,34 +137,34 @@ t key =
 
         -- \u{00A0} is a non-breaking space that prevents a single word wrapping to a new line
         Journey1Teaser ->
-            "He threw my phone on the floor and smashed\u{00A0}it"
+            "He considered my money\u{00A0}his"
```

Comment:
Since you pre-empted the move, you should archive this onto here: https://trello.com/c/mu7Hbodf/444-e-order-of-stories-top-should-be-about-money-as-phone-will-only-see-that-one

This might also cause conflict with my keywords branch :(

Reply:
I've put this back so you don't have conflicts